### PR TITLE
Feature/refactor into separate files

### PR DESCRIPTION
**Pull request by teenie-quaggard**

### CODE REVIEW

File: lib/formatter.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        @tile_content
+      elsif (last_column_tile(tile_number))
+        @tile_content << "\n" << row_divider() << "\n"
+      else
+        @tile_content << "|"
+      end
+    end
+
+    def last_tile(tile_number)
+      true if tile_number == @dimension**2
```
Comment (ellehallal) [inline lib/formatter.rb:21]: if this method contained `tile_number == @dimension**2`, what would it return?

File: lib/output.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class Output
+    attr_accessor :board
+    def initialize(args={})
+        @board = args[:board]
+    end
+
+    def prints_empty_board
```
Comment (ellehallal) [inline lib/output.rb:7]: What do you think about passing the board to this method via an argument?

File: lib/player.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class Player
```
Comment (ellehallal) [inline lib/player.rb:1]: Nice 👍🏽

**Review by ellehallal — COMMENTED** [review]
👍🏽 I've left a couple of comments. Give me a shout if you have any questions

File: lib/formatter.rb (hunk 1 above)
Comment (teenie-quaggard) [reply lib/formatter.rb:21]: Ah, of course - thank you Elle!

**Comment by teenie-quaggard** [comment]
Thanks for the review @ellehallal - I've pushed the changes. :)

**Pull request merged**